app/src/ble.c

```diff
@@ -163,9 +169,9 @@ int update_advertising() {
     struct bt_conn *conn;
     enum advertising_type desired_adv = ZMK_ADV_NONE;
 
-    if (active_profile_is_open() || !active_profile_is_connected()) {
+    if (active_profile_is_open() || !zmk_ble_active_profile_is_connected()) {
```

Comment:
Was the `!active_profile_is_connected()` intended here. It looks like that makes the `else if` case below dead code.

Reply:
Yeah, this was here due to me playing w/ the directed advertising. I think this can be removed and allow the below `else if` block handle the situation properly.